Cargo.toml

```diff
@@ -27,7 +26,6 @@ expect-test = "1.4.0"
 eyre = "0.6.8"
 notify-debouncer-mini = "0.2.1"
 ordered-float = "3.0"
-parking_lot = "0.12.1"
```

Comment:
I think `env_logger` can also be removed. It changes the output of one test because `cargo` wont suggest a type from the `env_logger` crate anymore but I'm not sure if it's work keeping a dependency just because of that.